lib/kernel_tag.rb

```diff
@@ -45,7 +45,7 @@ def kernel_match_version?(kernel_version, expected_kernel_versions)
   kernel_version = KernelTag.new(kernel_version)
 
   expected_kernel_versions.all? do |expected_kernel_version|
-    match = expected_kernel_version.match(/(?<operator>==|!=|<=|>|>=)?\s*(?<kernel_tag>v[0-9]\.\d+(?:\.\d+)?(?:-rc\d+)?)/)
+    match = expected_kernel_version.match(/(?<operator>==|!=|<|<=|>|>=)?\s*(?<kernel_tag>v[0-9]\.\d+(?:\.\d+)?(?:-rc\d+)?)/)
```

Comment:
thanks, is it possible not to change the code but only add spec so to cover its current behavior?